Argo/Globals/JSONDeserializer.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import Runes
+
+public struct JSONDeserializer {
+  public static func deserialize(data: NSData) -> JSON? {
```

Comment:
Fair. My thinking here is that it would at least indicate to consumers of this framework that they can implement additional deserializable types if they want to.